core/src/main/resources/application.yml

```diff
@@ -90,6 +90,7 @@ feast:
       provider: jwt
       options:
         jwkEndpointURI: "https://www.googleapis.com/oauth2/v3/certs"
+      disableRestControllerAuth: false
```

Comment:
Is this better to be just under `security`, instead of `security.authentication`?

Reply:
I think better to be under `security` since we bypass both authentication and authorization.